src/node.ts

```diff
@@ -765,6 +766,13 @@ export class Node implements IBehaviorAware<Node> {
         this._behaviors = [];
     }
 
+    protected _getBoudingInfoMin() {

```

Comment:
I don't understand why this is necessary.

Reply:
I moved getHierarchyBoundingVectors to node. But it didn't contain certain things that abstractMesh has like .subMeshes which is why I added these functions, in abstractMesh it overrides them to get the same behavior it originally had when called from an abstract mesh.